bin/update-oss-docs

```diff
@@ -1,6 +1,14 @@
 #!/bin/sh
 
-yarn global add susy sass-true sassdoc-theme-herman accoutrement --no-lockfile --ignore-scripts --no-bin-links --global-folder temp
+exit_on_error() {
+    exit_code=$1
+    if [ $exit_code -ne 0 ]; then
+        exit $exit_code
+    fi
+}
```

Comment:
You can avoid this whole function by adding ` -e` to the shebang line. That will make any command exiting with a nonzero code exit the whole script with that code.

Reply:
Yeah, I had that at first, but the decided I think we only want to bail if the first command fails. But I'm not really sure, maybe I should switch back.

Reply:
Not vital, and I understand that reasoning. Still 👍